task/common/values.go

```diff
@@ -26,7 +26,9 @@ type Status map[StatusCode]int
 type StatusCode string
 
 const (
-	StatusCodeRunning StatusCode = "running"
+	StatusCodeActive    StatusCode = "running"
+	StatusCodeSucceeded StatusCode = "succeeded"
```

Comment:
`success` sounds better and a more widely used work in APIs

Reply:
Tell that to Kubernetes developers. :smiling_imp: I'm being a bit facetious, of course, and agree that succeeded sounds a bit strange, but it's probably the only grammatically consistent option for that term. 😅

🔔 @casperdcl and your dictionary of synonyms.

Reply:
> but it's probably the only grammatically consistent option for that term. 

Also success